### PR TITLE
add April blog post

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

**Comment by dholbach** [comment]
Please review, but merge only EOD Wed 2021-03-31.

File: content/en/blog/2021-03-31-april-update.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+sub-modules, so we should be able to unblock Flux 1 users who use
+sub-modules instead of Kustomize remote git repositories.
+
+## Flux is a CNCF Incubation project
+
+![Flux in Incubation](/img/incubation.png)
+
+You will likely have seen the news elsewhere already, but Flux was
+promoted from CNCF Sandbox to CNCF Incubation. This is a huge step for
+the validation of our work, direction, en-user uptake and maturity of
```
Comment (squaremo) [inline content/en/blog/2021-03-31-april-update.md:88]:
```suggestion
the validation of our work, direction, end-user uptake and maturity of
```

File: content/en/blog/2021-03-31-april-update.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+- `flux create secret git` and `flux create source git` now support supplying a private key from a file using `--private-key-file`.
+- The `helm-controller` will now emit collected logs on release failures in the status conditions and as an event, this should make it much easier to debug wait timeout errors.
+- SOPS in the `kustomize-controller` has been updated to v3.7.0, support for the newly added age encryption format is planned. :hourglass_flowing_sand:
+- All controllers do now record the suspend status of resources in a gotk_suspend_status Prometheus gauge metric.
+
+🚀 Check out the guide on how to automate image updates to Git
+[https://toolkit.fluxcd.io/guides/image-update/](https://toolkit.fluxcd.io/guides/image-update/).
+
+Next up we will triage `image-*` issues and mark upcoming changes for
+v1alpha2.
```
Comment (squaremo) [inline content/en/blog/2021-03-31-april-update.md:73]:
```suggestion
v1alpha2. The proposal for v1alpha2 is under discussion at [https://github.com/fluxcd/flux2/discussions/1124](https://github.com/fluxcd/flux2/discussions/1124).
```

**Review by squaremo — COMMENTED** [review]
A couple of edits for you 📔

**Comment by stefanprodan** [comment]
@dholbach please add:

Flagger v1.7.0 is out! This release comes with support for manually approving the traffic weight increase. Starting with this version, Flagger can be used with Linkerd v2.10 and its new Viz addon, please see the updated [guide](https://docs.flagger.app/tutorials/linkerd-progressive-delivery). Thanks to the Linkerd team for contributing to Flagger.

**Pull request merged**